lib/src/json_schema/validator.dart

```diff
@@ -113,13 +113,19 @@ class Validator {
 
   /// A private constructor for recursive validations.
   /// [inEvaluatedItemsContext] and [inEvaluatedPropertiesContext] are used to pass in the parents context state.
-  Validator._(this._rootSchema, {bool inEvaluatedItemsContext = false, bool inEvaluatedPropertiesContext = false}) {
+  Validator._(this._rootSchema,
+      {bool inEvaluatedItemsContext = false,
+      bool inEvaluatedPropertiesContext = false,
+      Map<JsonSchema, JsonSchema> initialDynamicParents}) {
```

Comment:
Does anything ever pass something for this? I'm probably just Dart-illiterate so I'm not seeing it.